README.md

```diff
@@ -287,6 +287,37 @@ Kubeclient::Client.new(
 ```
 
 
+#### Amazon EKS Credentials
+
+On Amazon EKS, when using aws-iam-authentication, with [credentials](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/Credentials.html),
```

Comment:
Can you clarify whether it's for cluster running on EKS, or the code calling Kubeclient, or both?

Reply:
i'll put a blurb in there.  it's possible to use aws-iam-authentication with kubernetes clusters which are self-hosted on EC2.  I've never tested these tokens in that configuration.